app/views/health/careplans/_pctp_signatures.haml

```diff
@@ -1,31 +1,62 @@
 %article.health-care-plan__responsible-parties.l-print__section--page
   %h1 Responsible Parties Signatures
 
-  %dl.l-print__item
-    %dt.inline Patient:
-    %dd.inline= @patient.name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.patient_signed_on&.to_date
+  .signature__section
+    .signature__section__row
+      .signature__label Patient:
+      .signature__value= @patient.name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Patient Signature:
+        .signature__line
+      .signature__section__signedOn
+        .signature__label Signed On:
+        - if @careplan.patient_signed_on&.to_date
+          .signature__value= @careplan.patient_signed_on&.to_date
+        - else
+          .signature__line
 
-  %dl.l-print__item
-    %dt.inline Representative:
-    %dd.inline= @careplan.representative&.full_name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.representative_signed_on
+  .signature__section
+    .signature__section__row
+      .signature__label Representative:
+      - if @careplan.representative&.full_name
+        .signature__value= @careplan.representative&.full_name
+      - else
+        .signature__line.signature__line-name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Representative Signature:
+        .signature__line
+      .signature__section__signedOn
+        .signature__label Signed On:
+        - if @careplan.representative_signed_on
+          .signature__value= @careplan.representative_signed_on
+        - else
+          .signature__line
 
-  %dl.l-print__item
-    %dt.inline Provider:
-    %dd.inline
-      - if @careplan.provider_signed_on.present?
-        = @careplan.provider&.full_name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.provider_signed_on&.to_date
-  %strong
-    -if  @careplan.member_verbalizes_understanding
-      _X_
-    - else
-      ___
-  Member verbalizes understanding and accepts care plan. (Note: signature needed within 3 mos. of verbal approval.)
+  .signature__section
+    .signature__section__row
+      .signature__label Provider:
+      - if @careplan.provider&.full_name && @careplan.provider_signed_on.present?
+        .signature__value= @careplan.provider&.full_name
+      - else
+        .signature__line.signature__line-name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Provider Signature:
+        .signature__line
+      .signature__section__signedOn
+        .signature__label Signed On:
+        - if @careplan.provider_signed_on&.to_date
+          .signature__value= @careplan.provider_signed_on&.to_date
+        - else
+          .signature__line
+
+  .signature__section.signature__understanding
+    %strong
+      -if @careplan.member_verbalizes_understanding
+        _X_
```

Comment:
This is tiny, but I think we have the ability to use `yes_no(true)` now, with the actual checkmark.  Can we use that instead of `_X_`?

Reply:
Like this?

![Screen Shot 2022-09-28 at 11 48 38 AM](https://user-images.githubusercontent.com/5333982/192825919-426744f5-9e25-495b-ad05-40a95ec7c582.png)

Reply:
works for me